modules/core/shared/src/main/scala/lucuma/core/model/sequence/InstrumentConfig.scala

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2016-2022 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.model.sequence
+
+import cats.Eq
+import cats.syntax.all._
+import lucuma.core.enum._
+import monocle.Focus
+import monocle.Lens
+import monocle.Prism
+import monocle.macros.GenPrism
+import org.typelevel.cats.time._
+
+import java.time.Duration
+
+sealed trait InstrumentConfig
+object InstrumentConfig {
```

Comment:
The static config is also instrument config, just unchanging from step to step.  I guess that's why I called this the dynamic config as opposed to the static config.  I'm not opposed to this name, but I would like to know your thoughts on the change.

Reply:
I'm still trying to grasp what everything means TBH. In this case I took the name from the property where it's used in the API. If I understand correctly, it would be better suited to call it `DynamicConfig` ?

Reply:
In the API it is down in the step. Every step has a dynamic instrument config and I think I chose `instrumentConfig` to distinguish it from the non-instrument step configuration (gcal setup, offset position, etc.). Maybe I should rename it `dynamicConfig` there too.

Reply:
Ok, for the moment I left `instrumentConfig: DynamicConfig`. LMK if you plan to change it so we name it accordingly here.